src/playground/components/Configuration.js

```diff
@@ -228,7 +228,7 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
                                 </h4>
                                 <input
                                     id={ruleName}
-                                    className={rulesWithInvalidConfigs.has(ruleName) ? "config__added-rules__rule-input-error" : ""}
+                                    className={rulesWithInvalidConfigs.has(ruleName) || validationError?.message.includes(ruleName) ? "config__added-rules__rule-input-error" : ""}
```

Comment:
This might alert rules that happen to be substrings of the invalid one

![image](https://user-images.githubusercontent.com/44349756/173130646-8bc0df72-d74c-4dec-85ad-54fdc734c1be.png)

Maybe we could add quotes from the error message?

```suggestion
                                    className={rulesWithInvalidConfigs.has(ruleName) || validationError?.message.includes(`"${ruleName}"`) ? "config__added-rules__rule-input-error" : ""}
```

Reply:
Good catch, thanks! Resolved.